PublicationsRouter/sword-in/USER.md

```diff
@@ -21,7 +26,7 @@ There will only be one collection - this will be the collection for your user ac
 
 You can get the URL for your collection by retrieving the service document, which you can do like this:
 
-    GET /
+    GET /sword/
```

Comment:
Are you saying that the URL is: `https://pubrouter.jisc.ac.uk/sword/sword/` ?

Which is what I might construe in relation to line 12 and line 29:
`https://pubrouter.jisc.ac.uk/sword`
`GET /sword/`